Modules/rat_command.py

```diff
@@ -51,6 +52,12 @@ class NameCollisionException(CommandException):
 # character/s that must prefix a message for it to be parsed as a command.
 prefix = config['commands']['prefix']
 
+# Pydle bot instance.
+bot: BasicClient = None
+
+# facts manager instance
+facts_manager: FactsManager = FactsManager()
```

Comment:
Methinks this should be created in MechaClient?

Reply:
It doesn't really matter, as every constructor call will return the same instance #singleton

Reply:
It unnecessarially introduces another instance of the facts manager, which will make testing harder.

As it is already provided as part of the `MechaClient` instance, it is provided in the `context` argument of all command executions. 

During testing we should only need to monkeypatch this object in one place, if at all.

by having it at the module level in rat_command as well as an instance-attribute of the Mechaclient, it creates two places where a patch may be necessary which makes testing potentially more difficult